### DISCUSSION TITLE
Eject External Drive

### DISCUSSION THREAD

**Discussion by ohana54** [johnlindquist/kit#772] [Share]
[Open eject-external-drive in Script Kit](https://scriptkit.com/api/new?name=eject-external-drive&url=https://gist.githubusercontent.com/ohana54/ad6c3269ccc036daaec2d7036260404a/raw/32c800da6ea9871bd99f06e5867d3648118d9db6/eject-external-drive.js")

```js
// Name: Eject External Drive

import "@johnlindquist/kit"
import os from 'os'

const {stdout} = await $`diskutil list | grep "(external"`

const firstDisk = stdout.split(os.EOL)[0]
const firstDiskPath = firstDisk.split(' ')[0]

await $`diskutil eject ${firstDiskPath}`

```

**Comment by johnlindquist** [comment]
Awesome, thanks so much for sharing!
